src/main/java/org/zalando/fahrschein/domain/DataOperation.java

```diff
@@ -0,0 +1,28 @@
+package org.zalando.fahrschein.domain;
+
+/**
+ * The type of operation executed on the entity.
+ * <ul>
+ * <li><strong>C</strong>: Creation</li>
+ * <li><strong>U</strong>: Update</li>
+ * <li><strong>D</strong>: Deletion</li>
+ * <li><strong>S</strong>: Snapshot</li>
+ * </ul>
+ */
+enum DataOperation {
```

Comment:
Sure, I can add the `@JsonProperty` to the interface. Should I do the same for
- getDataType
- getOperation
- getMetadata

I could also add `@JsonCreator` to the Metadata constructor.